### ISSUE TITLE
ID attributes should be unique

### ISSUE THREAD

**Issue by dannychapman** [department-of-veterans-affairs/vets-website#862]
https://www.vets.gov/beta/2015/11/05/why-we-are-designing-in-beta.html
https://www.vets.gov/disability-benefits/claims-appeal/court-of-appeals/veterans-appeals-board/index.html
https://www.vets.gov/disability-benefits/claims-appeal/court-of-appeals/
https://www.vets.gov/disability-benefits/claims-appeal/court-of-appeals/veterans-appeals-board/	
https://www.vets.gov/disability-benefits/claims-appeal/court-of-appeals/court-appeals-veterans-claims/
